developer-docs/upgrade/index.md

```diff
@@ -4,147 +4,126 @@ page_title: Build and Deploy
-| Create Upload_CollectionHierarchy_CSV jenkins job                                                                 | [PR Link](https://github.com/project-sunbird/sunbird-devops/pull/2990)                                                                 |
-| Upgrade postgres to version 11 in vdn staging                                                                     | Follow below steps [Link](https://project-sunbird.atlassian.net/wiki/spaces/DevOps/pages/3008462870/Generic+Postgres+Upgrade+Document) |
-| Please update the Question Paper collection category definition <br> in Dock staging	Category Definition details | Ref: https://project-sunbird.atlassian.net/browse/SB-26908                                                                             |
-| Category defination updates in VDN                                                                                | Ref: https://project-sunbird.atlassian.net/wiki/spaces/SingleSource/pages/3014164481/Content+NOT+created+under+a+target+collection     |
-
+| Systemd Changes | "We've to make manual changes, to avoid reprovisioning of services. Folllow the steps to make those changes. Follow these instructions for the services mentioned below<br><br>- [ ] Systemctl: <br>      Follow the instructions for the services one node at a time<br>      - open service File (file path is mentioned as part of service details below) and add following line under `[Service]`<br>        ```bash<br>        Environment=""""_JAVA_OPTIONS='-Dlog4j2.formatMsgNoLookups=true'""""<br>        ```<br>      - After adding that line <br>        ```bash<br>        sudo systemctl daemon-reload<br>        sudo systemctl restart <proper service name><br>        sudo systemctl status <proper service name> # Should be greeen<br>        ```<br>  - [ ] Elasticsearch (* noted because es service name changes as per the host name. Check the actual service name in the `/usr/lib/systemd/system` folder)<br>        Service Name: *elasticsearch.service <br>        File: /usr/lib/systemd/system/*elasticsearch.service<br>  - [ ] cassandra <br>        Service Name: cassandra.service<br>        File: /run/systemd/generator.late/cassandra.service<br>  - [ ] Keycloak <br>        Service Name: keycloak.service<br>        File: /run/systemd/generator.late/keycloak.service<br>  - [ ] graylog <br>        Service Name: graylog-server.service<br>        File: /usr/lib/systemd/system/graylog-server.service<br>  - [ ] redis <br>        Service Name: redis.service<br>        File: /etc/systemd/system/redis.service<br>  - [ ] mongo <br>        Service Name: mongod.service<br>        File: /lib/systemd/system/mongod.service<br>  - [ ] jenkins <br>        Service Name: jenkins.service<br>        file: /run/systemd/generator.late/jenkins.service" |  |
```

Comment:
@rjshrjndrn is the word 'Systemd' correct? checking so that it is not a spelling error.

Reply:
Yeah, it's a linux service.